swarm/fuse/swarmfs_test.go

```diff
@@ -351,16 +344,18 @@ func (ta *testAPI) unmountWhenResourceBusy(t *testing.T) {
 
 	actualPath := filepath.Join(testMountDir, "2.txt")
```

Comment:
It was failing because an error was ignored at https://github.com/ethereum/go-ethereum/pull/14951/files#diff-01b0c465879e1dc77de7f53d797cd350R347 and nothing past that error was failing.